.github/workflows/govulncheck-action.yml

```
@@ -0,0 +1,52 @@
+name: Build & Test 'govulncheck-action'
+on:
+  push:
+    branches:
+    - master
+    tags-ignore:
+    - '*.*'
+    paths:
+    - 'govulncheck-action/**'
+  pull_request:
+    branches:
+    - master
+    paths:
+    - 'govulncheck-action/**'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    name: Test
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: govulncheck-action/go.mod
+
+    - name: Cache dependencies
+      uses: actions/cache@v4
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles ('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+          
+    - name: Lint
+      uses: golangci/golangci-lint-action@v8
+      with:
+        version: v2.1.6
+        skip-pkg-cache: true
+        skip-build-cache: true
+        args: --config=./govulncheck-action/.golangci.yml --verbose
```

related to my comment in the other PR https://github.com/codeready-toolchain/toolchain-cicd/pull/142#discussion_r2174933585
```suggestion
        args: --config=./govulncheck-action/.golangci.yml --verbose ../govulncheck-action/...
```